### DISCUSSION TITLE
Practica 2 HTML Pastelería - Debemos seguirnos todos en GitHub

### DISCUSSION THREAD

**Discussion by RIvanCF** [Launch-X-Latam/MisionFrontEnd#408] [Show and tell]
Les doy a conocer el enlace del proyecto de la pastelería. 
Vaya que si le metí muchísimo tiempo. 
La adapte para versión desktop y movile así que  también es compatible en celulares. 
Use un dominio gratuito que Netlify proporciona. 
Pero en mi repo también se encuentra el dominio GitHub Page
El link de mi repositorio es la segunda imagen, me ayudarían mucho si le dedican una estrellita. 
Tambien propongo que todos nosotros nos sigamos en GitHub y de ser posible, en nuestras redes sociales para no perder contacto 
Los veo en la cima compañeros EXPLORERS 👨🏻‍🚀
  
<a href="https://bright-puffpuff-95a437.netlify.app/index.html"><img src="https://img.shields.io/badge/Practica%20HTML-Netfily-blue" height="30"></a>

<a href="https://github.com/RIvanCF/LaunchXLATAM_S2"><img src="https://img.shields.io/github/stars/RIvanCF/LaunchXLATAM_S2?style=social" height="30"></a>

**Comment by GonDev1** [comment]
Muy buena iniciativa, te sigo ✌
